Comment out definitions of functions that are part of Glib.

--- pinentry-0.7.5/gtk+-2/gtksecentry.c	2007-11-19 12:20:50.000000000 +0100
+++ pinentry-0.7.5/gtk+-2/gtksecentry.c	2008-12-08 22:08:22.000000000 +0100
@@ -269,6 +269,7 @@ gboolean g_use_secure_mem = FALSE;
 				} while(0)
 
 
+#if 0
 gpointer
 g_malloc(gulong size)
 {
@@ -347,6 +348,7 @@ g_free(gpointer mem)
 	    free(mem);
     }
 }
+#endif
 
 GType
 gtk_secure_entry_get_type(void)
